conf/messages

```diff
@@ -74,7 +74,7 @@ applicationsuccessful.banner.heading=Application successful
 applicationsuccessful.banner.subheading=Your LISA and ISA manager references are the same:
 applicationsuccessful.heading=What this means for you
 applicationsuccessful.text.line1.pre=You can use your software to connect to the
-applicationsuccessful.text.line1.link=Lifetime ISA API
+applicationsuccessful.text.line1.link=Lifetime ISA API (opens in a new window or tab)
```

Comment:
O in Open should be caps as per user story

Reply:
Updated